src/components/popover-menu/popover-menu.tsx

```diff
@@ -0,0 +1,107 @@
+import {A11yWrapper} from '@playkit-js/common';
+import {h, Component, VNode} from 'preact';
+
+const {withEventManager} = KalturaPlayer.ui.Event;
+const {ENTER, ESC, SPACE} = KalturaPlayer.ui.utils.KeyMap;
+
+import * as styles from './popover-menu.scss';
+
+interface PopoverMenuItemData {
+  testId: string;
+  label: string;
+  onClick: () => void;
+}
+
+interface PopoverMenuProps {
+  eventManager?: any;
+  children?: VNode;
+  items: Array<PopoverMenuItemData>;
+}
+
+interface PopoverMenuState {
+  isOpen: boolean;
+}
+
+@withEventManager
+class PopoverMenu extends Component<PopoverMenuProps, PopoverMenuState> {
+  private _controlElementRef: HTMLDivElement | null = null;
+  private _popoverElementRef: HTMLDivElement | null = null;
+
+  eventManager: any;
+
+  constructor(props: PopoverMenuProps) {
+    super(props);
+    this.state = {isOpen: false};
+
+    this.props.eventManager?.listen(document, 'click', this.handleMouseEvent);
```

Comment:
I believe listener can be added only if user opened popover (and removed once popover got closed)